packages/code-du-travail-frontend/src/outils/DureePreavisDemission/steps/Result.js

```diff
@@ -3,16 +3,26 @@ import PropTypes from "prop-types";
 import { Toast } from "@socialgouv/react-ui";
 
 import { SectionTitle } from "../../common/stepStyles";
-import { ccCriterionName, filterSituations, recapSituation } from "./situation";
+import {
+  filterSituations,
```

Comment:
filterSituation / getSituationFor should be in utils too

Reply:
They're already are on utils but exported within situation